sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/changefeed/implementation/ChangeFeedStartFromInternal.java

```diff
@@ -37,6 +37,10 @@ public static ChangeFeedStartFromInternal createFromPointInTime(Instant pointInT
         return new ChangeFeedStartFromPointInTimeImpl(pointInTime);
     }
 
+    public static ChangeFeedStartFromInternal createFromLegacyContinuation() {
```

Comment:
where this is being used?

Reply:
SparkBridgeImplementationInternal.scala line 284

Reply:
thanks, somehow missed it